### PR TITLE
Support large file uploading

### PR DESCRIPTION
**Pull request by HunseopJeong**
- Upload file via formidable
- Change the zip extraction module to the 'extract-zip'
- Apply ES6 to the 'import.js'
- Add progress dialog

Signed-off-by: Hunseop Jeong <hs85.jeong@samsung.com>

### CODE REVIEW

**Comment by HunseopJeong** [comment]
I hope to extract zip file using the file name.
But the 'decompress-unzip' module used buffer.
So I changed the module.

File: public/css/style.css [hunk 1]
```diff
@@ -81,4 +81,21 @@ a.list-group-item.active small {
...
   height: 300px;
   overflow: auto;
+}
+
+@keyframes spin {
+  0% { transform: rotate(0deg); }
+  100% { transform: rotate(360deg); }
+}
+
+.progress-circle {
```
Comment (Kwanghyuk-Kim) [inline public/css/style.css]: did you run the csslint for this modification ?

File: public/js/project.js [hunk 2]
```diff
@@ -428,43 +428,73 @@ $('#import-project').on('show.bs.modal', function() {
...
+      var formData = new FormData();
+      formData.append('uploads[]', file, file.name);
+
+      $.ajax({
+        url: '/import/archive',
+        type: 'put',
+        data: {
+          name: projectName,
+          filename: file.name,
+          description: projectDescription,
```
Comment (Kwanghyuk-Kim) [inline public/js/project.js]: end of this line, comma ?

File: public/css/style.css (hunk 1 above)
Comment (HunseopJeong) [reply public/css/style.css]: Fixed.

File: public/js/project.js (hunk 2 above)
Comment (HunseopJeong) [reply public/js/project.js]: Fixed.

**Pull request merged**